openmc/settings.py

```diff
@@ -652,7 +653,7 @@ def surf_source_write(self, surf_source_write: dict):
         cv.check_type('surface source writing options', surf_source_write, Mapping)
         for key, value in surf_source_write.items():
             cv.check_value('surface source writing key', key,
-                           ('surface_ids', 'max_particles'))
+                           ('surface_ids', 'max_particles', 'mcpl'))
```

Comment:
You're obviously right, but it would become irrelevant if an mcpl-check is implemented in the c++-layer for output as well, to keep the xml-format unchanged. I have no preference either way.